### ISSUE TITLE
Unable to include libimobiledevice-win32 in release mode in vc++

### ISSUE THREAD

**Issue by akhilritu** [storoj/libimobiledevice-win32#23]
Unable to include libimobiledevice-win32 in release mode in vc++ because it is giving a lot of linking errors.

**Comment by mirekp** [comment]
yes. There are a lot of issues in release. You need to update include definitions and change a few compiler settings in order to build it in release.

**Comment by akhilritu** [comment]
Hello Sir,
please help me to update include definitions

**Comment by akhilritu** [comment]
Hello Sir,
Please help me to update include definitions.

**Comment by akhilritu** [comment]
Hello mirekp sir,
How can i update include definitions and change compiler settings in order to build it in release.
Please help.....
Can i build it in vc++ 2010.
